src/components/base/JAssetSymbol/jAssetSymbol.scss

```diff
@@ -45,51 +72,80 @@
+    &.-symbol-kcs { background-image: $token-kcs-blue; }
     &.-symbol-knc { background-image: $token-knc-blue; }
+    &.-symbol-link { background-image: $token-link-blue; }
+    &.-symbol-lrc { background-image: $token-lrc-blue; }
     &.-symbol-lun { background-image: $token-lun-blue; }
     &.-symbol-mana { background-image: $token-mana-blue; }
     &.-symbol-mco { background-image: $token-mco-blue; }
+    &.-symbol-mgo { background-image: $token-mgo-blue; }
     &.-symbol-mkr { background-image: $token-mkr-blue; }
     &.-symbol-mln { background-image: $token-mln-blue; }
     &.-symbol-mtl { background-image: $token-mtl-blue; }
+    &.-symbol-nas { background-image: $token-nas-blue; }
     &.-symbol-net { background-image: $token-net-blue; }
     &.-symbol-nmr { background-image: $token-nmr-blue; }
+    &.-symbol-npxs { background-image: $token-npxs-blue; }
     &.-symbol-null { background-image: $token-null-blue; }
     &.-symbol-omg { background-image: $token-omg-blue; }
     &.-symbol-pay { background-image: $token-pay-blue; }
     &.-symbol-plu { background-image: $token-plu-blue; }
+    &.-symbol-poly { background-image: $token-poly-blue; }
+    &.-symbol-powr { background-image: $token-powr-blue; }
+    &.-symbol-ppt { background-image: $token-ppt-blue; }
+    &.-symbol-prl { background-image: $token-prl-blue; }
+    &.-symbol-qash { background-image: $token-qash-blue; }
     &.-symbol-qtum { background-image: $token-qtum-blue; }
     &.-symbol-rep { background-image: $token-rep-blue; }
+    &.-symbol-rhoc { background-image: $token-rhoc-blue; }
     &.-symbol-rlc { background-image: $token-rlc-blue; }
     &.-symbol-san { background-image: $token-san-blue; }
     &.-symbol-sngls { background-image: $token-sngls-blue; }
     &.-symbol-snt { background-image: $token-snt-blue; }
     &.-symbol-storj { background-image: $token-storj-blue; }
     &.-symbol-swt { background-image: $token-swt-blue; }
     &.-symbol-taas { background-image: $token-taas-blue; }
+    &.-symbol-theta { background-image: $token-theta-blue; }
     &.-symbol-time { background-image: $token-time-blue; }
     &.-symbol-tkn { background-image: $token-tkn-blue; }
     &.-symbol-trst { background-image: $token-trst-blue; }
+    &.-symbol-trx { background-image: $token-trx-blue; }
+    &.-symbol-ven { background-image: $token-ven-blue; }
     &.-symbol-vsl { background-image: $token-vsl-blue; }
+    &.-symbol-wax { background-image: $token-wax-blue; }
     &.-symbol-wings { background-image: $token-wings-blue; }
     &.-symbol-xaur { background-image: $token-xaur-blue; }
+    &.-symbol-xin { background-image: $token-xin-blue; }
+    &.-symbol-xpa { background-image: $token-xpa-blue; }
+    &.-symbol-zil { background-image: $token-zil-blue; }
     &.-symbol-zrx { background-image: $token-zrx-blue; }
   }
 
   &.-gray {
-    background-image: linear-gradient(to top, $dusk, $dusk);
+    &.-symbol-not-listed { color: $blue; }
```

Comment:
`{ color: $dusk; }` ?

Reply:
Gray text is invisible on white background :(